modules/product/src/main/java/com/opengamma/strata/product/etd/EtdOptionSecurity.java

```diff
@@ -207,22 +265,34 @@ public EtdContractSpecId getContractSpecId() {
    * This is used to describe the instance of the option.
    * @return the value of the property, not null
    */
-  public YearMonth getExpiry() {
-    return expiry;
+  public YearMonth getExpiryMonth() {
```

Comment:
I'm not sure this change is an improvement. The value is a year-month so calling it "month" is slightly misleading. When I see `getExpiryMonth` I expect it to return something like `Month.MARCH`, not a `YearMonth`.